assets/src/blocks/Covers/CoversEditor.js

```diff
@@ -0,0 +1,153 @@
+import {
+  SelectControl,
+  PanelBody
+} from '@wordpress/components';
+import { useEffect } from '@wordpress/element';
+
+import { InspectorControls } from '@wordpress/block-editor';
+import TagSelector from '../../components/TagSelector/TagSelector';
+import PostSelector from '../../components/PostSelector/PostSelector';
+import PostTypeSelector from '../../components/PostTypeSelector/PostTypeSelector';
+import { Covers, COVER_TYPES } from './Covers';
+import { getCoversClassName } from './getCoversClassName';
+import { useCovers } from './useCovers';
+
+const { RichText } = wp.blockEditor;
+const { __ } = wp.i18n;
+
+const renderEdit = (attributes, toAttribute) => {
+  const { initialRowsLimit, posts, tags, cover_type, post_types } = attributes;
+
+  const rowLimitOptions = [
+    { label: __('1 Row', 'planet4-blocks-backend'), value: 1 },
+    { label: __('2 Rows', 'planet4-blocks-backend'), value: 2 },
+    { label: __('All rows', 'planet4-blocks-backend'), value: 0 },
+  ];
+
+  return (
+    <InspectorControls>
+      <PanelBody title={__('Setting', 'planet4-blocks-backend')}>
+        <SelectControl
+          label='Rows to display'
+          value={initialRowsLimit}
+          options={rowLimitOptions}
+          onChange={value => toAttribute('initialRowsLimit')(Number(value))}
+        />
+
+        {!posts.length &&
+          <div>
```

Comment:
Do we still need these divs around the component now it's in the sidebar? :thinking:

Reply:
For this one we do actually, because of the help paragraph below... But I'll remove the one around `PostTypeSelector`!